### ISSUE TITLE
Input values cant be copied

### ISSUE THREAD

**Issue by prajaktagadhave123** [WordPress/gutenberg#13761]
In setting option Copy, Cut, Paste not working. It copies blocks HTML.

Here are steps to reproduce this issue - 
1) Add a paragraph or any cor block.
2) Click on the input field and copy that value - `https://monosnap.com/file/PB41q8nKRvCKdTIkcGmoUbCUcT0OCa`
3) Then add this copied value in any editor - it gives HTML of the block. - `https://monosnap.com/file/lpJtkp2ysaIrYfq5tlnyY1USfkzlcY`

**Comment by youknowriad** [comment]
I can't reproduce personally but the instructions are not very clear, can you share a screenshot?

**Comment by prajaktagadhave123** [comment]
Steps-

1 - Add a paragraph to your page. 
2 - Click on Forn setting option form settings. add any random values in Fornt size text box. Here I nused 45. - 
Refere this screenshot - https://monosnap.com/file/PB41q8nKRvCKdTIkcGmoUbCUcT0OCa
3 - Select & copy values from this Font size text box - https://monosnap.com/file/GSdKaNSCix8uKMw9qZ1Tr5TIf087K2
4 - Paste this values in your text editor - https://monosnap.com/file/FCiZucaszQuqhm5KxCocmrq3tm9uiC
Press CTR+V (paste this values i any text editor it give output of your HTML block)

Here is recording for you - https://monosnap.com/file/yFZ0Ud2ruVVqKk79RvhWt9bpb2Nm0i

Hope now you are able to reproduce this issue.

Ideal situation-

If we copied text from input i.e(45) then it shoul paste 45 not your block's HTML.
